node/app.js

```diff
@@ -109,6 +109,8 @@ io.sockets.on('connection', function (socket, player) {
                 [],
                 []
               ]
+
+              socket.ready = 0;
```

Comment:
Is this of any use for this particular PR? It doesn't seem to. If not, remove it.